lib/libfrr.c

```diff
@@ -61,7 +61,7 @@ static char pidfile_default[512];
 #ifdef HAVE_SQLITE3
 static char dbfile_default[512];
 #endif
-static char vtypath_default[256];
+static char vtypath_default[512];
```

Comment:
looked like this only gets used with `snprintf` - was there some other issue with its original size?

Reply:
the warning was that the theoretical sizing of vtypath_default and the string sizes of other char's used could cause an overrun.  Not likely to really happen but just making it 512 makes the problem go away.